documentation/src/main/resources/pages/ditto/protocol-specification.md

```diff
@@ -78,6 +78,7 @@ There are some pre-defined headers, which have a special meaning for Ditto:
 | `requested-acks` | Defines which [acknowledgements](basic-acknowledgements.html) are requested for a command processed by Ditto. | `JsonArray` of `String` - default: `["twin-persisted"]` |
 | `timeout` | Defines how long the Ditto server should wait, e.g. applied when waiting for requested acknowledgements. | `String` - e.g.: `42s` or `250ms` or `1m` - default: `60s`|
 | `version` | Determines in which schema version the `payload` should be interpreted. | `Number` - currently: \[1,2\] |
+| `metadata` | Determines which metadata information is stored in the thing. | `JsonArray` of `String` - default: `` |
```

Comment:
Is the data type correct? It's actually JsonArray of at least Object? Maybe MetadataEntry? It's a complex datatype after all

Reply:
You are absolutely right. @thjaeckle can you comment on the exact type. I have to few knowledge about dittos type system :/

Reply:
First, "metadata" is the wrong header key. It must be "put-metadata".
The data type is JsonArray of JsonObjects containing a "key" with the value being a JsonString and a "value" with the value being a JsonValue (any valid json value should work here, also arrays, objects).